engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/JobPreparationActor.scala

```diff
@@ -9,85 +9,56 @@ import wdl4s.expression.WdlStandardLibraryFunctions
 import wdl4s.util.TryUtil
 import wdl4s.values.WdlValue
 
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.Future
 import scala.util.{Failure, Success, Try}
 
 object JobPreparationActor {
-  /** Commands */
-  case object Start
-  /** Responses */
-  case class BackendJobPreparationSucceeded(jobDescriptor: BackendJobDescriptor, props: Props)
-  case class BackendJobPreparationFailed(jobKey: JobKey, throwable: Throwable)
-  /** States */
-  sealed trait JobPreparationActorState
-  case object Idle extends JobPreparationActorState
-  case object EvaluatingInputs extends JobPreparationActorState
-  case object BuildingProps extends JobPreparationActorState
-  /** Internal Messages*/
-  private case class EvaluatedInputs(inputs: Map[LocallyQualifiedName, WdlValue])
-  private case class BackendActorProps(jobDescriptor: BackendJobDescriptor, props: Props)
+  sealed trait JobPreparationActorCommands
+  case object Start extends JobPreparationActorCommands
+
+  sealed trait JobPreparationActorResponse
+  case class BackendJobPreparationSucceeded(jobDescriptor: BackendJobDescriptor, props: Props) extends JobPreparationActorResponse
+  case class BackendJobPreparationFailed(jobKey: JobKey, throwable: Throwable) extends JobPreparationActorResponse
 
   def props(executionData: WorkflowExecutionActorData,
             jobKey: BackendJobDescriptorKey,
             factory: BackendLifecycleActorFactory,
-            configDescriptor: BackendConfigurationDescriptor) = Props(new JobPreparationActor(executionData, jobKey, factory, configDescriptor))
+            configDescriptor: BackendConfigurationDescriptor) = {
+    Props(new JobPreparationActor(executionData, jobKey, factory, configDescriptor)).withDispatcher("akka.dispatchers.slow-actor-dispatcher")
```

Comment:
That's pretty much what I was on about w/ the enhanced class on Props. Since you don't seem to care a lot and my opinion is waffling back and forth the final outcome will probably depend on what I happen to think when I go to merge this in ;)